### PR TITLE
[vcpkg-scripts] z_vcpkg_fixup_macho_rpath_in_dir: Call install_name_tool only once

### PR DESCRIPTION
**Pull request by autoantwort**
Should probably merged at the same time as #39889 to reduce would rebuilds

### CODE REVIEW

File: scripts/ports.cmake [hunk 1]
```diff
@@ -196,7 +196,7 @@ target system or to the host system. Use a prefixed variable instead.
             z_vcpkg_fixup_rpath_in_dir()
         endif()
         if(VCPKG_FIXUP_MACHO_RPATH OR (VCPKG_TARGET_IS_OSX AND NOT DEFINED VCPKG_FIXUP_MACHO_RPATH))
-            z_vcpkg_fixup_macho_rpath_in_dir()
+            z_vcpkg_fixup_macho_rpath_in_dir() # v2
```
Comment (autoantwort) [inline scripts/ports.cmake:199]: Necessary to get another ABI hash

Comment (Osyotr) [reply scripts/ports.cmake:199]: https://github.com/microsoft/vcpkg/blob/f37740d06373970a4bd29a366475797695192644/scripts/ports.cmake#L27-L28

Comment (autoantwort) [reply scripts/ports.cmake:199]: Since there is no semantic change in this PR I will simply remove the change in this file

**Review by m-kuhn — APPROVED** [review]

**Review by data-queue — APPROVED** [review]

**Pull request merged**